Marlin/src/pins/lpc1768/pins_BTT_SKR_V1_1.h

```diff
@@ -52,6 +52,14 @@
 #define E0_DIR_PIN                         P2_13
 #define E0_ENABLE_PIN                      P2_12
 
+#define E1_STEP_PIN                        P0_01
+#define E1_DIR_PIN                         P0_00
+#define E1_ENABLE_PIN                      P0_10
```

Comment:
The E1 pins are already defined in `pins_BTT_SKR_common.h`. You shouldn't need to redefine them for any of these SKR boards.

Reply:
you are right. Sorry for the pull request.